src/bundle/Controller/SectionController.php

```diff
@@ -128,17 +155,66 @@ public function viewAction(Section $section): Response
             new SectionDeleteData($section)
         )->createView();
 
+        return $this->render('EzPlatformAdminUiBundle:admin/section:view.html.twig', [
+            'section' => $section,
+            'form_section_delete' => $sectionDeleteForm,
+            'deletable' => !$this->sectionService->isSectionUsed($section),
+            'can_edit' => $this->isGranted(new Attribute('section', 'edit')),
```

Comment:
Hmm, I see this is not the first time you use it here, so question to @Nattfarinn @webhdx: how does this work? 
Does it take into policy limitations? How is our permission system tied to Symfony firewall?
I rather expected usage of `PermissionResolver` for this.

Reply:
Ping @webhdx @Nattfarinn